deploy/vagrant/provision/roles/spark/files/dist.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-tar czf /tmp/spark.tar -C /spark/dist .
-rm -rf /spark/*
-tar xzf /tmp/spark.tar -C /spark
-rm -f /tmp/spark.tar
```

Comment:
Is this what was deleting everything? And make distribution is actually fine to use?